pkg/exporter/syslogexporter/README.md

```diff
@@ -26,7 +24,10 @@ __Note__: Syslog over UDP doesn't support certificate verification (`ca_certific
 - `format` - rfc5424/rfc3164
   - `rfc5424` - Expects the syslog messages to be rfc5424 compliant
   - `rfc3164` - Expects the syslog messages to be rfc3164 compliant
-- `ca_certificate` [tcp only] - A publicly verifiable server certificate (`note`: Self signed certificates are not supported in this version)
+- `ca_certificate` [tcp only] - A publicly verifiable server certificate
+- `certificate` [tcp only] - certificate for mTLS communication (client certificate)
+- `key` [tcp only] -  Key for mTLS communication (client key)
+- `skip_certificate_verification` - [tcp only] If set to `true` then any certificate presented by the server is accepted and any host name in that certificate
```

Comment:
Can we group configuration under common key?

Reply:
I will try to use https://github.com/open-telemetry/opentelemetry-collector/blob/7cf999e4526c9e65f5dbedb67c90cbe2c79e4f54/config/configtls/configtls.go#L66 and make settings similar to configuration described here: https://github.com/open-telemetry/opentelemetry-collector/blob/exporter/otlphttpexporter/v0.71.0/config/configtls/README.md